src/FakeItEasy/Configuration/FakeConfigurationManager.cs

```diff
@@ -61,6 +59,123 @@ public IAnyCallConfigurationWithNoReturnTypeSpecified CallTo(object fakeObject)
             return this.configurationFactory.CreateAnyCallConfiguration(manager, rule);
         }
 
+        public IPropertySetterConfiguration<TValue> CallToSet<TValue>(Expression<Func<TValue>> propertySpecification)
+        {
+            Guard.AgainstNull(propertySpecification, nameof(propertySpecification));
+            this.AssertThatMemberCanBeIntercepted(propertySpecification);
+
+            var setterExpression = BuildSetterFromGetter(propertySpecification);
+
+            return new PropertySetterConfiguration<TValue>(setterExpression, this.CreateVoidArgumentValidationConfiguration);
+        }
+
+        private static string GetIndexerName(MethodCallExpression methodCallExpression)
+        {
+            if (methodCallExpression != null &&
+                HasThis(methodCallExpression) &&
+                methodCallExpression.Method.IsSpecialName)
+            {
+                var methodName = methodCallExpression.Method.Name;
+                if (methodName.StartsWith("get_", StringComparison.Ordinal))
+                {
+                    return methodName.Substring(4);
+                }
+            }
+
+            return null;
+        }
+
+        private static bool HasThis(MethodCallExpression methodCallExpression)
+        {
+            return (methodCallExpression.Method.CallingConvention & CallingConventions.HasThis) ==
+                   CallingConventions.HasThis;
+        }
+
+        private static MethodCallExpression BuildSetterFromMemberExpression<TValue>(MemberExpression memberExpression)
+        {
+            PropertyInfo propertyInfo = memberExpression.Member as PropertyInfo;
+            if (propertyInfo == null)
+            {
+                throw new ArgumentException("The expression refers to '" + memberExpression.Member.Name +
+                                            "', which is a field, not a property getter.");
+            }
+
+            var propertySetterInfo = propertyInfo.GetSetMethod(nonPublic: true);
+            if (propertySetterInfo == null)
+            {
+                throw new ArgumentException("The property '" + memberExpression.Member.Name + "' does not have a setter.");
+            }
+
+            var arguments = new[] { BuildArgumentThatMatchesAnything<TValue>() };

```

Comment:
> … and for whatever reason, the overloads all showed up for me today. I swear last night I saw 3, and today 5. I wonder what I was really typing…

Probably because the current project was FakeItEasy.Net35. You can't use this method, it's not available in 3.5.